idaes/models/properties/activity_coeff_models/activity_coeff_prop_pack.py

```diff
@@ -1800,3 +1826,88 @@ def default_material_balance_type(self):
 
     def default_energy_balance_type(self):
         return EnergyBalanceType.enthalpyTotal
+
+    def calculate_scaling_factors(self):
+        # Get default scale factors and do calculations from base classes
+        super().calculate_scaling_factors()
+
+        phases = self.params.config.valid_phase
+        is_two_phase = len(phases) == 2  # possibly {Liq}, {Vap}, or {Liq, Vap}
+        sf_flow = iscale.get_scaling_factor(
+            self.flow_mol_phase_comp, default=1, warning=True
+        )
+        sf_T = iscale.get_scaling_factor(self.temperature, default=1, warning=True)
+        sf_P = iscale.get_scaling_factor(self.pressure, default=1, warning=True)
+
+        if self.is_property_constructed("_teq"):
+            iscale.set_scaling_factor(self._teq, sf_T)
+        if self.is_property_constructed("_teq_constraint"):
+            iscale.constraint_scaling_transform(
+                self._teq_constraint, sf_T, overwrite=False
+            )
+
+        if self.is_property_constructed("_t1"):
+            iscale.set_scaling_factor(self._t1, sf_T)
+        if self.is_property_constructed("_t1_constraint"):
+            iscale.constraint_scaling_transform(
+                self._t1_constraint, sf_T, overwrite=False
+            )
+
+        if self.is_property_constructed("total_flow_balance"):
+            s = iscale.get_scaling_factor(self.flow_mol, default=1, warning=True)
+            iscale.constraint_scaling_transform(
+                self.total_flow_balance, s, overwrite=False
+            )
+
+        if self.is_property_constructed("component_flow_balances"):
+            for i, c in self.component_flow_balances.items():
+                if is_two_phase:
+                    s = iscale.get_scaling_factor(
+                        self.mole_frac_comp[i], default=1, warning=True
+                    )
+                    s *= sf_flow
+                    iscale.constraint_scaling_transform(c, s, overwrite=False)
+                else:
+                    s = iscale.get_scaling_factor(
+                        self.mole_frac_comp[i], default=1, warning=True
+                    )
+                    iscale.constraint_scaling_transform(c, s, overwrite=False)
+
+        if self.is_property_constructed("density_mol"):
+            for c in self.eq_density_mol.values():
+                iscale.constraint_scaling_transform(c, sf_P, overwrite=False)
```

Comment:
Why is molar density being scaled using the pressure scaling factor?

Reply:
dens_mol_phase from the cubic prop pack was scaled using the pressure scaling factor, so I just carried it over w/o really double-checking it. But looking at it now, it doesn't make much intuitive sense why this would be the case, so I'll change it.